docs/source/posts/2021/2021-06-21-week-3-sajag.rst

```diff
@@ -0,0 +1,49 @@
+Second week of coding!
+======================
+
+.. post:: June 21 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Welcome to the third weekly check-in. I'll be sharing my progress for the second week of coding.
+
+What did you do this week?
+--------------------------
+I created an example to demonstrate how one can render multiple bonds (double and triple). This required me to write an algorithm to detect bonding.
+I used `this blog <https://www.kaggle.com/aekoch95/bonds-from-structure-data>`_ as a reference and made a few tweaks of my own to detect the presence of double/triple bonds from interatomic distances.
+The math involved in generating the coordinates of bonds was quite intriguing. Preview:
+  
+  .. figure:: https://user-images.githubusercontent.com/65067354/122672109-7d040c80-d1e7-11eb-815d-1d07fe47bbc4.png
+    :width: 300
+    :height: 300
+
+    molecules rendered: Ethane, Ethene, Ethyne (from left to right)
+
+In addition to this, I tried understanding the codebase of vtkMolecule, vtkSimpleBondPerceiver, vtkMoleculeMapper, vtkPeriodicTable and was able to render bond-stick models and stick models using it.
+This will be of great help although it's rather slow in rendering large molecules (using shaders to improve its speed will be crucial if it's to be utilised).
```

Comment:
typo: utilized

Reply:
Utilise is correct, it's the British spelling.

Reply:
Went with an online spell check tool, the creator must not be British😆